src/MovieGrid/MovieGrid.js

```diff
@@ -13,6 +13,13 @@ class MovieGrid extends Component {
 
   async componentDidMount() {
     const allMovies = await ApiCalls.getAllMovies()
+    if(this.props.currentUserId) {
```

Comment:
Good call on moving this into MovieGrid so that we could pass the data into MovieView properly.